src/vm/exceptionhandling.cpp

```diff
@@ -4833,7 +4833,11 @@ DWORD64 GetModRMOperandValue(BYTE rex, BYTE* ip, PCONTEXT pContext, bool is8Bit,
             // Check for RIP-relative addressing mode.
```

Comment:
Could you please update the comment to reflect the fact that for x86 it is not RIP-relative addressing mode?